Content.Client/Theta/ModularRadar/Modules/ShipEvent/RadarShieldStatus.cs

```diff
@@ -0,0 +1,39 @@
+using System.Numerics;
+using Content.Shared.Theta.ShipEvent.Components;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+
+namespace Content.Client.Theta.ModularRadar.Modules.ShipEvent;
+
+public sealed class RadarShieldStatus : RadarModule
+{
+    private readonly IEntityManager entMan = default!; 
+    private readonly TransformSystem formSys = default!;
+    
+    public RadarShieldStatus(ModularRadarControl parentRadar) : base(parentRadar)
+    {
+        entMan = IoCManager.Resolve<IEntityManager>();
+        formSys = entMan.System<TransformSystem>(); //no idea why it can't resolve it
```

Comment:
```suggestion
        formSys = entMan.System<TransformSystem>();
```
Так делают и коллеги. Почему? ХЗ. А вот на энтити менеджер можешь кинуть {Dependency]

Reply:
окей уберу коммент. но депенденси это лишняя строчка

Reply:
Это лишние {Dependency] букв, а у тебя лишнаяя строчка

Reply:
так в данном случае ещё и айокманагер.инджектдепенденсис() надо писать кроме тега. ну и неоднородно, не хочу крч

Reply:
айокманагер в родительком конструкторе вызывается